### PR TITLE
feat: added Toast component into CL

### PR DESCRIPTION
**Pull request by Manju-sree**
Added Toast component.
Toast_gif

### CODE REVIEW

File: Components/Toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+.simple-toast-body {
+    padding: 1rem;
+    background-color: #E5E7EB;
```
Comment (Bhakti199) [inline Components/Toast/toast.css]: use color variables

File: Components/Toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    border-radius: 5px;
+    color: black;
```
Comment (Bhakti199) [inline Components/Toast/toast.css:11]: here too

File: Components/Toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+.toast-success {
+    background-color: #70f764;
+}
+
+.toast-error {
+    background-color: #ec5752;
+}
+
+.toast-warning {
+    background-color: #f8bb64;
```
Comment (Bhakti199) [inline Components/Toast/toast.css]: here same as above

File: Components/Toast/toast.html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                    <div class="toast-body">
+                        Can't send photos. Retry in 5 seconds.
+                        <button class="retry-btn">RETRY</button>
+                        <span class="icon-toast">
+                            <i class="fas fa-times"></i>
+                        </span>
+                    </div>
+                </div>
+                <div>
+                    <iframe src="https://www.thiscodeworks.com/embed/62037b93234b8a0015700c85" style="width: 100%; height: 107px;" frameborder="0"></iframe>
```
Comment (Bhakti199) [inline Components/Toast/toast.html:85]: for height use rem units

**Review by Bhakti199 — COMMENTED** [review]
nicely written code, suggested some changes.

File: Components/Toast/toast.html (hunk 4 above)
Comment (Manju-sree) [reply Components/Toast/toast.html:85]: for iframe width and height is given default by thiscodeworks , i'm not changing it

Comment (Bhakti199) [reply Components/Toast/toast.html:85]: okay

File: Components/Toast/toast.css (hunk 3 above)
Comment (Manju-sree) [reply Components/Toast/toast.css]: done changes..

**Review by Bhakti199 — APPROVED** [review]
good to merge

**Review by mariyasada — APPROVED** [review]

File: Components/Toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+.retry:hover {
+    background-color: hsla(0, 0%, 100%, 0.1);
+}
+
+.icon-toast {
+    margin-left: 0.6rem;
+    margin-top: -0.7rem;
+    border-radius: 50%;
+    cursor: pointer;
+    padding: 0.3rem;
```
Comment (fahd23) [inline Components/Toast/toast.css:85]: any reason to use rem here

File: Components/Toast/toast.css [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+    font-family: "Times New Roman", Times, serif;
+}
+
+
+/* example of toast */
+
+.toast-body {
+    position: relative;
+    display: flex;
+    padding: 18px 12px;
```
Comment (fahd23) [inline Components/Toast/toast.css:40]: you can use rem here

**Review by fahd23 — APPROVED** [review]
Looks good

**Pull request merged**